src/animation.js

```diff
@@ -12,9 +12,9 @@
 //     See the License for the specific language governing permissions and
 // limitations under the License.
 
-(function(scope, testing) {
+(function(shared, scope, testing) {
 
-  var sequenceNumber = 0;
+  shared.sequenceNumber = 0;
```

Comment:
> OK, not a big fan of it, but it's small and if it's useful for debugging I guess it's OK.

I don't think I fully understand the repercussions of it being shared. What's the difference between that and it being in WAN?